zillow_address_pull.py

```diff
@@ -93,11 +177,11 @@ def zillow_info(address, zipcode):
     # add neighborhoods to df
     housing_data = pd.merge(
         housing_data
-        , neighborhood_mapping
+        # , neighborhood_mapping
         , left_on='zip_code'
         , right_on='zip_code'
         , how='left'
     )
 
     # return the values for the Flask App
-    return zestimate, rent_zestimate, home_size, num_of_bedrooms, num_of_bathrooms, property_size, year_built
+    return zestimate, rent_zestimate, home_size, num_of_bedrooms, num_of_bathrooms, property_size, year_built, last_sold_date, last_sold_price
```

Comment:
It seems like it makes more sense to have any function that queries the underlying dataset be of the same return type. So both `random_address` and `zillow_info` should probably return the exact same structs.

Also I think it would help to wrap these in a dictionary or `namedtuple` of some kind so that downstream users can access fields they care about in a future-proof way. As it currently stands, if you want to add more fields to this then any downstream users that unpack the tuple will have to always refactor to handle a longer tuple.